packages/server/modules/core/services/users.js

```diff
@@ -73,25 +73,17 @@ module.exports = {
     const usr = await userByEmailQuery(user.email).select('id').first()
     if (usr) throw new Error('Email taken. Try logging in?')
 
-    const res = await Users().returning('id').insert(user)
+    await Users().insert(user)
+    const newUser = await module.exports.getUserById({ userId: newId })
```

Comment:
Why do we need to query back the user? It feels like we do not trust, that if the line above succeeds, we have the user in the DB...
The users.insert should return a failure if it cannot create the user...

Reply:
fixed https://github.com/specklesystems/speckle-server/pull/967/commits/f003270b5b6b5aa2e2fa113b19a1714eaaf67e16